File: backend/StudioManagement.Data/Entities/Service.cs
namespace StudioManagement.Data.Entities;

public class Service : ITenantEntity
{
    public int ServiceId { get; set; }
    public int StudioId { get; set; }
    public string ServiceName { get; set; } = null!;
    public string? Description { get; set; }
    public decimal DefaultPrice { get; set; }
    public bool IsActive { get; set; } = true;
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }

    public Studio Studio { get; set; } = null!;
    public ICollection<QuotationItem> QuotationItems { get; set; } = new List<QuotationItem>();
}

File: src/types/service.ts
export interface StudioService {
  serviceId: number;
  serviceName: string;
  description: string | null;
  defaultPrice: number;
  isActive: boolean;
  createdAt: string;
  updatedAt: string;
}

export interface CreateServiceRequest {
  serviceName: string;
  description?: string;
  defaultPrice: number;
}

export type UpdateServiceRequest = CreateServiceRequest;
